### ISSUE TITLE
Enabling restic DEBUG_LOG breaks backrest

### ISSUE THREAD

**Issue by Thinkscape** [garethgeorge/backrest#316]
**Describe the bug**
Setting `DEBUG_LOG=/tmp/restic-debug.log` as env var breaks restic.

**To Reproduce**

1. Go to repo settings
2. In env vars add `DEBUG_LOG=/tmp/restic-debug.log`
3. Run backup.

**Expected behavior**
I get a debug log in /tmp and restic completes backup.

**Actual**
```
error: failed to get snapshots for plan: get snapshots for plan "backup": command "/bin/restic-0.16.4 snapshots --json -o sftp.args=-oBatchMode=yes --tag plan:backup,created-by:nas.local" failed: command output is not valid JSON: invalid character 'd' looking for beginning of value
Output:
debug log file /tmp/restic-debug.log
```
<img width="417" alt="image" src="https://github.com/garethgeorge/backrest/assets/270528/bba8c660-58c7-4c52-9269-c6d8bd57d34c">

**Platform Info**

- Linux, AMD64
- backrest 1.0.0

**Comment by garethgeorge** [comment]
Definitely needs thinking, assuming this is related to debugging https://github.com/garethgeorge/backrest/issues/315 re: hung operations! Backrest should definitely support this sort of debug-ability. 

Right now backrest specifically asserts that the first output from restic _must_ be JSON, but perhaps the right assertion is simply that the last thing output by restic must be a summary event. If it isn't then restic can include all non-JSON output it got in an error message it generates.

The handling is here, https://github.com/garethgeorge/backrest/blob/main/pkg/restic/outputs.go#L101-L116 I'm traveling at the moment so it'll be ~1w before I can get a patch release out. In the immediate term you can work around this with a custom build by deleting those lines (or trying a PR to fix it which I'm happy to look at!).

**Comment by garethgeorge** [comment]
Fixed in 1.2.0 and above.
